src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -1,9 +1,18 @@
 package com.checkscanner.check;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.springframework.boot.autoconfigure.EnableAutoConfiguration'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)